presto-native-execution/README.md

```diff
@@ -84,9 +84,7 @@ A reminder of the available Makefile targets can be obtained using `make help`
 ```
 
 ## Build using Dockerfile
-Run `make runtime-container` in the presto-native-execution root directory
-to build run-ready containerized version of Prestissimo. Information on available
-configuration options can be found in [scripts/release-centos-dockerfile/README.md](scripts/release-centos-dockerfile/README.md)
+Check [Jenkinsfile](../Jenkinsfile) for detailed build steps.
```

Comment:
Not many are aware of the Jenkinsfile layout and semantics.
Can we explicitly layout the steps here?
Most use-cases here would be to build all the three stages.

Reply:
It was not clear to me which part of the Jenkinsfile needed to be followed.